### ISSUE TITLE
Is this still supported? 

### ISSUE THREAD

**Issue by givihuda** [leokhoa/laragon#309]
The last release is May '21, version 5.0.0.

I was hoping there'd be a Mac version.

**Comment by leokhoa** [comment]
Yes, it is still supported. With Laragon, you don't need to update regularly.

**Comment by givihuda** [comment]
OK, well, I consider the need to update regularly sort of relative, though generally it's a rule since software is constantly changing.

So I'm glad that it's still supported, but considering it's been at 5.0.0 since May of '21, is the repo expected to have any bug fixes or improvements?

**Comment by ajcastro** [comment]
Hi @leokhoa  Is PHP 8.1 officially supported by Laragon?

**Comment by alxndr-w** [comment]
Have a look at <https://github.com/leokhoa/laragon/discussions/categories/announcements>
